setup.py

```diff
@@ -19,7 +19,7 @@ def check_for_openmp():
     #include <stdio.h>
 
     int main(void) {
-        #pragma omp parallel
+        #pragma omp parallel num_threads(2)
```

Comment:
What does this do?

Reply:
It makes the parallel block spawn two threads regardless of the number of cores. Without it, OpenMP would not be enabled if there is only one logical processor available or `OMP_NUM_THREADS=1` has been set in the environment.
On Windows, prior to this change the code would print one line on my quad-core CPU and detect OpenMP as working even though it wasn't.